enterprise_access/apps/api/v1/views/subsidy_access_policy.py

```diff
@@ -566,22 +591,43 @@ def can_redeem(self, request, enterprise_customer_uuid):
                         "reason": reason,
                         "policy_uuids": [policy.uuid for policy in policies],
                     })
+
+            # Select one redeemable policy (i.e. use the "policy resolver") and serialize it for output.
+            resolved_policy = None
             if redeemable_policies:
                 resolved_policy = SubsidyAccessPolicy.resolve_policy(redeemable_policies)
-                serialized_policy = serializers.SubsidyAccessPolicyRedeemableResponseSerializer(resolved_policy).data
 
+            # Determine if the learner has already redeemed the requested content_key.
             has_successful_redemption = any(
                 redemption['state'] == TransactionStateChoices.COMMITTED
                 for redemption in redemptions
             )
-            can_redeem_for_content_response = {
+
+            # Determine the price for content for display purposes only.
+            try:
+                content_metadata = self.subsidy_client.get_subsidy_content_data(enterprise_customer_uuid, content_key)
+                # Note that the "content_price" key is guaranteed to exist, but the value may be None.  Also, the
+                # upstream field type is a string representing a decimal dollar.
+                list_price_decimal_dollars_str = content_metadata["content_price"]
+                list_price_integer_cents = int(float(list_price_decimal_dollars_str) * 100)
```

Comment:
[curious] Is it worthwhile to return both the integer cents and the decimal dollars? It feels a bit odd for the API to convert the list price to cents only for the caller of the API (learner portal) to convert it back to decimal dollars. A potential idea:

```js
{
  "list_price": {
    "USD": 299,
    "USD_CENTS": 29900,
  }
}
```

Reply:
Related, locally, I'm seeing a course price of $299.00 in course-discovery be returned as `299000` in the `can-redeem` response, which would equate to `$29,900`.

It looks like `get_subsidy_content_data` is already returning integer cents?

Reply:
I think I would agree about being explicit about the type, since the unit in this case doesn't directly pertain to any unit of any particular ledger/subsidy.

Reply:
updated to match your proposed format.  Note that `usd` is a float, and `usd_cents` is an integer.